aten/src/ATen/native/native_functions.yaml

```diff
@@ -1784,3 +1784,10 @@
 
 - func: get_device(Tensor self) -> int64_t
   device_guard: False
+
+- func: cpu(Tensor self) -> Tensor
+  variants: method
+
+# TODO: Would be preferable if this were an at::optional<int64_t> device
```

Comment:
We need to fix optional on Windows first (believe me, I tried really hard and had to change it all back to -1 in the end...)

Reply:
Actually I don't know if you wanted the `optional` here to represent the default device as `nullopt`, or to support the logic of `dispatch_type_conversion`. We would need it for the latter.